academy-transfers-api.sln

```diff
@@ -34,7 +34,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Helpers.Tests", "Helpers.Te
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Frontend.Integration.Tests", "Frontend.Integration.Tests\Frontend.Integration.Tests.csproj", "{E0C8BA1B-0620-45B1-A7ED-87CBC6BE21C6}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Data.Tests", "Data.Tests\Data.Tests.csproj", "{83CC61DF-CA4C-472A-B3CC-A3C2CE8EFBF5}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Data.Tests", "Data.Tests\Data.Tests.csproj", "{83CC61DF-CA4C-472A-B3CC-A3C2CE8EFBF5}"
```

Comment:
Why has this changed?

Reply:
Couldn't tell you honestly but its randomly generated so can't imagine it's an issue?